armor-write/pom.xml

```diff
@@ -37,5 +37,10 @@
             <artifactId>fastutil</artifactId>
             <version>8.3.0</version>
         </dependency>
+        <dependency>
```

Comment:
We want to avoid using guava, if necessary. Just so it doesn't collide with other version of guava.